site/content/docs/5.0/components/scrollspy.md

```diff
@@ -12,7 +12,7 @@ Scrollspy has a few requirements to function properly:
 
 - It must be used on a Bootstrap [nav component]({{< docsref "/components/navs" >}}) or [list group]({{< docsref "/components/list-group" >}}).
 - Scrollspy requires `position: relative;` on the element you're spying on, usually the `<body>`.
-- When spying on elements other than the `<body>`, be sure to have a `height` set and `overflow-y: scroll;` applied.
+- When spying on elements other than the `<body>`, be sure to have a `height` set and `overflow-y: scroll;` applied—alongside a `tabindex="0"` to ensure keyboard access.
```

Comment:
i'd even consider splitting out the entire bullet into a separate callout, giving a bit more context about "If you're making a scrollable container..."

Reply:
That made sense to me to add it on the same sentence than the one specifying how to make a scrollable container.
I'll try another bullet, let me know what you think :)

Reply:
no sorry i meant the entire bullet (not just your additional point) there could be ripped out into a callout (as it's not necessarily about scrollspy per se, but more about "make sure you do these things when making a scroller")

Reply:
Oh ok, I misunderstood!

Let me try this, then. That makes sense.